src/library/autodj/dlgautodj.ui

```diff
@@ -115,16 +115,26 @@
       </widget>
      </item>
      <item>
-      <widget class="QSpinBox" name="spinBoxTransition">
-       <property name="focusPolicy">
-        <enum>Qt::ClickFocus</enum>
+      <widget class="QComboBox" name="ComboBoxAutoDjTransitionUnit">
+       <property name="toolTip">
+        <string/>
        </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QComboBox" name="ComboBoxAutoDjCalcTransitionPeriod"/>
+     </item>
```

Comment:
How about instead of adding this QComboBox we make separate options for the transition unit combobox:
  * Seconds
  * Beats (outgoing track)
  * Beats (incoming track)

Reply:
Makes sense, I'll push an update with those changes.